src/main/kotlin/com/exactpro/th2/http/client/api/Th2RawHttpRequest.kt

```
@@ -56,4 +57,12 @@ class Th2RawHttpRequest(
         parentEventId,
         metadataProperties
     )
+
+    override fun eagerly(): EagerHttpRequest {
+        error("Unsupported eagerly call of request. It's client side, no need to eager request")
```

It's better to throw UnsupportedOperationException instead